test/cpp/ext/otel/otel_plugin_test.cc

```diff
@@ -813,12 +812,15 @@ TEST_F(OpenTelemetryPluginOptionEnd2EndTest, ClientOnlyPluginOption) {
   const auto& server_attributes =
       data["grpc.server.call.duration"][0].attributes.GetAttributes();
   EXPECT_EQ(server_attributes.size(), 2);
-  EXPECT_EQ(server_attributes.find("key"), server_attributes.end());
+  EXPECT_THAT(
+      server_attributes,
+      ::testing::Not(::testing::Contains(std::make_pair("key", "value"))));
```

Comment:
We want to check that the key is not present, even if it has a different value.  Please change this to `::testing::Contains(::testing::Key("key"))`.

Same thing throughout.

Reply:
Done